src/Widgets/calendar/ControlHeader.vala

```diff
@@ -19,35 +19,85 @@
 
 
 namespace DateTime.Widgets {
-    public class ControlHeader : Gtk.Box {
+    public class ControlHeader : Gtk.Grid {
         public signal void left_clicked ();
         public signal void right_clicked ();
         public signal void center_clicked ();
+
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+        }
+
+        construct {
+            string style = """
```

Comment:
Can you put custom styles in a .css file and load with gresource please?

Reply:
Can do.